app/api/events.py

```diff
@@ -74,7 +74,11 @@
     'timezone': fields.String(),
     'latitude': fields.Float(),
     'longitude': fields.Float(),
-    'background_image': fields.Upload(attribute='background_url'),
+    'background_image': {
```

Comment:
@poush I don't think you define nested attributes like this. I believe you need to use `fields.Nested`. Not very sure though. Could you check Flask-RESTPlus docs and confirm ?

Also, please check and fix the travis build fail.